libr/core/cbin.c

```diff
@@ -2883,6 +2883,10 @@ static int bin_classes(RCore *r, int mode) {
 			r_list_foreach (c->methods, iter2, sym) {
 				char *mflags = r_core_bin_method_flags_str (sym->method_flags, mode);
 				char *cmd = r_str_newf ("\"f method%s.%s.%s = 0x%"PFMT64x"\"\n", mflags, c->name, sym->name, sym->vaddr);
+				r_str_replace_char (cmd, ' ', '_');
+				cmd[2] = ' ';
+				char *eq = strrchr (cmd, '=');
```

Comment:
use r_str_rchr to be portable

Reply:
ok 0898dfc6fbaac57bf88ec21bb133d940fc38133f